polaris-react/src/components/ResourceList/ResourceList.stories.tsx

```diff
@@ -20,7 +20,7 @@ export default {
 
 export function Default() {
   return (
-    <Card padding="0">
+    <Card padding="0" roundedAbove="sm">
```

Comment:
Added `roundedAbove="sm"` prop to ResourceList stories to match styling on IndexTable.